src/unread/__tests__/unreadSelectors-test.js

```diff
@@ -83,9 +48,7 @@ describe('getUnreadByStream', () => {
           in_home_view: true,
         },
       ],
-      unread: {
-        streams: unreadStreamData,
-      },
+      unread: unreadState,
```

Comment:
Separate from this refactor, I think there's something a bit wrong with the description for this test: 'when there are unread stream messages, returns a list with counts per stream_id '.

The thing returned isn't a list, it's an object-as-map.

Reply:
Hmm indeed. I'd guess that the description was true when originally written, and then the data structure was improved :slightly_smiling_face: ... huh, nope, `git log -S` on that description finds 81a481e76 where it was already wrong in just this way.

Anyway, I'll fix it.